src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -100,34 +100,38 @@ class ClosedLoans extends React.Component {
     };
 
     this.sortMap = {
-      [this.columnMapping.title]: loan => _.get(loan, ['item', 'title']),
-      [this.columnMapping.barcode]: loan => _.get(loan, ['item', 'barcode']),
-      [this.columnMapping['Fee/Fine']]: loan => this.getFeeFine(loan),
-      [this.columnMapping.loanDate]: loan => loan.loanDate,
-      [this.columnMapping.dueDate]: loan => loan.dueDate,
-      [this.columnMapping['Call Number']]: loan => _.get(loan, ['item', 'callNumber']),
-      [this.columnMapping.Contributors]: loan => {
+      title: loan => _.get(loan, ['item', 'title']),
```

Comment:
@MaksymDryha This change here in closedLoans currently breaks the sorting functionality on the individual columns.  We use this.columnMapping as the single source of all the translations, and use the same for sorting purposes as well. Any reason why we should be moving away from this? Refer to my PR https://github.com/folio-org/ui-users/pull/613 for further info, and also https://issues.folio.org/browse/STCOM-399 for issue with MCL header localization. Thanks.